tests/refund_test.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+import easypost
+import tests.fixture as fixture
+
+
+@pytest.mark.vcr()
+def test_refund_create():
+    shipment = easypost.Shipment.create(**fixture.one_call_buy_shipment())
+
+    retrieved_shipment = easypost.Shipment.retrieve(
+        shipment
+    )  # We need to retrieve the shipment so that the tracking_code has time to populate
```

Comment:
Comments like this may read better if they came just before the code (that way they aren't formatted weird due to `Black`)